qa/integration/fixtures/kafka_input_spec.yml

```diff
@@ -0,0 +1,24 @@
+---
+services:
+  - logstash
+  - kafka
+config: |-
+  input {
+    kafka {
+      topics => "logstash_topic_plain"
+      auto_offset_reset => "earliest"
+      codec => "plain"
+      group_id => "ls10"
+    }
+  }
+  output {
+    #file {
+    #  path => "kafka_input.output"
+    #  flush_interval => 0
+    #  codec => line { format => "%{message}" }
+    #}
+  }
+
```

Comment:
Since the `stdout` is now fixed for the new concurrency model we can now is it.
Maybe in a future PR we could add a new helper that check the input vs the file output and check it they match?